example/advection/advection_package.cpp

```diff
@@ -40,8 +40,6 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   pkg->AddParam<>("vx", vx);
   Real vy = pin->GetOrAddReal("Advection", "vy", 1.0);
```

Comment:
I don't know where these changes came from.  I haven't touched this file intentionally.  What is the best path forward to revert it?  Or should I let it be as is?